### ISSUE TITLE
Unable to set up Push Notification - Error: Unknown error. Please contact support@flutterflow.io

### ISSUE THREAD

**Issue by rgrajivchn** [FlutterFlow/flutterflow-issues#2253]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Unable to set up Push Notification - - Error: Unknown error. Please contact support@flutterflow.io

Error during push notifications deployment

App - gospelin-chennai-7xv73z

### Expected Behavior

Push notifications should have been deployed properly

### Steps to Reproduce

App Settings --> Push notifications --> Deploy

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4CkcmBx4h2ofVA1pSJc8JFmiUXHEw/RYEZ1utDQgIcJ5PMC+wIZ/f7XFRuTde0YwlEOk6KoDkdpqDemvOSMMBeIQidba5R/pB1Ww/vQlO6VarTO6W0Zm5/DcBUFWqz4bW3uyNQB/pvSXwb1zqAJtm/bDfsGL7NNm0CDc6lNZCK2SrDX1iXc2URm05KZDPz

### Context

Unable to set up Push Notification - - Error: Unknown error. Please contact support@flutterflow.io

Error during push notifications deployment

### Visual documentation

gospelin-chennai-7xv73z

### Environment

```markdown
- FlutterFlow version: 4.1
- Platform: web
- Browser name and version: Chrome
- Operating system and version affected: Windows 11
```

**Comment by takashiff** [comment]
Hello @rgrajivchn, without further information, it may be difficult for us to find the root cause of the issue. Please contact our technical support team at support@flutterflow.io for further assistance on resolving this issue.

**Comment by rgrajivchn** [comment]
What further information?, your system gave only that one line errorRajivSent from my iPadOn 01-Feb-2024, at 5:34 AM, takashiff ***@***.***> wrote:﻿
Hello @rgrajivchn, without further information, it may be difficult for us to find the root cause of the issue. Please contact our technical support team at ***@***.*** for further assistance on resolving this issue.

**Comment by rgrajivchn** [comment]
How can this be closed without fixing the bugSent from my iPadOn 01-Feb-2024, at 5:34 AM, takashiff ***@***.***> wrote:﻿
Closed #2253 as completed.
